test/e2e/fixture-builder.js

```diff
@@ -1257,16 +1252,16 @@ class FixtureBuilder {
                 },
               ],
               date: 1708029792962,
+              id: 'oKXoF_MNlffiR2u1Y3mDE',
```

Comment:
Nit: Maybe we can revert these re-order changes to minimize the diff

Reply:
done here https://github.com/MetaMask/metamask-extension/pull/27847/commits/651a40c659663e8ff1c82c4108d9eb4b1e59b333